doc/src/nimbleparse.md

```diff
@@ -1,49 +1,177 @@
 # nimbleparse
 
-If you have a [`lrlex`](lrlex.html) compatible `.l` file and an
-[`lrpar`](lrpar.html) compatible `.y` file, you can use `nimbleparse` as a quick
-way of testing inputs and exploring the resulting parse tree:
+`nimbleparse` is a simple grammar debugging aid. It takes as input a Lex
+specification, a Yacc specification, and an input file and prints any warnings
+about the specifications (e.g. shift/reduce errors) as well as the resulting
+parse tree to stdout. If the parse is unsuccessful it will report parsing
+errors and, when possible fixes. If parsing is successful, `nimbleparse` exits
+with 0; if an error is detected it exits with 1.
+
+The full command-line specification is as follows:
 
 ```
-$ cargo build --release
-$ target/release/nimbleparse -h
-Usage: nimbleparse [-r <cpctplus|mf|panic|none>] [-y <eco|original>] <lexer.l> <parser.y> <input file>
+nimbleparse [-r <cpctplus|none>] [-y <eco|grmtools|original>] [-q] <lexer.l> <parser.y> <input file>
```

Comment:
`mf` isn't actually gone yet, right? So we are just not documenting it so that people don't use it?

What about `panic`?

Reply:
> mf isn't actually gone yet, right? So we are just not documenting it so that people don't use it?

Correct.

> What about panic?

That *might* disappear at some point. I certainly don't want people to rely on its existence at the moment.